src/store/reducers/loginReducer.ts

```diff
@@ -6,6 +6,8 @@ import {
 } from '../actions/login';
 
 export type UserType = {
+  isDefault?: any;
+  selected?: any;
```

Comment:
Add types please, try to avoid any. You can make this types as temporary any, in future it will be easier to find this types and fix